src/encoded/schemas/treatment_concentration_series.json

```diff
@@ -26,7 +26,7 @@
         }
     },
     "facets": {
-        "related_datasets.assay_term_name": {
+        "assay_term_name": {
```

Comment:
Yes, assay_term_name should be calculated in treatment_concentration_series and that should be what goes into the facet, not digging into related_datasets. All the other series should also have this in their schemas